Tests/Common/Background Queue/QueueRunnerTest.swift

```diff
@@ -0,0 +1,36 @@
+@testable import CioInternalCommon
+import Foundation
+import SharedTests
+import XCTest
+
+class QueueRunnerTest: UnitTest {
+    private var runner: CioQueueRunner!
+
+    private let httpClientMock = HttpClientMock()
+
+    override func setUp() {
+        super.setUp()
+
+        runner = CioQueueRunner(jsonAdapter: jsonAdapter, logger: log, httpClient: httpClientMock, hooksManager: diGraph.hooksManager, sdkConfig: sdkConfig)
+    }
+
+    // MARK: registerPushToken
+
+    func test_registerPushToken_givenIdentifierEmpty_expectHttpRequestNotMade_expectBQDeletesTask() {
```

Comment:
can we have a test for the `failure` of the `result` as well?

Reply:
Ah, good call-out. 

I added a function to Swift's built-in Result class. Looking at the diff, I do see that I didn't name the file correctly and I did not write tests for the added function. 

I just pushed a commit taking care of those things. Thanks for bringing that up.